src/main/java/com/internet/shop/controller/IndexController.java

```diff
@@ -1,11 +1,13 @@
-package com.internet.shop.controllers;
+package com.internet.shop.controller;
 
 import java.io.IOException;
 import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+@WebServlet("/")
```

Comment:
can we use one type of mapping? annotation or web.xml?

Reply:
I have left only annotations